packages/expo-cli/src/commands/info/diagnosticsAsync.ts

```diff
@@ -1,50 +0,0 @@
-import { getDefaultTarget } from '@expo/config';
-import envinfo from 'envinfo';
```

Comment:
Remove `envinfo` from the `expo-cli/package.json` and commit the `yarn.lock` changes if any...